pkg/api/v1/schema.go

```diff
@@ -158,6 +158,12 @@ type CPU struct {
 	// Cores specifies the number of cores inside the vmi.
 	// Must be a value greater or equal 1.
 	Cores uint32 `json:"cores,omitempty"`
+	// Sockets specifies the number of sockets inside the vmi.
+	// Must be a value greater or equal 1.
+	Sockets uint32 `json:"sockets,omitempty"`
```

Comment:
...especially considering that you are using (correctly, seeing the surrounding code) unsigned integers here.